CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# dbt_shopify_source v0.11.0
+[PR #78](https://github.com/fivetran/dbt_shopify_source/pull/78) introduces the following changes: 
+
+## 🚨 Breaking Changes 🚨
+- Added `source_relation` to the `partition_by` clauses that determine the `is_most_recent_record` in the `stg_shopify__metafield` and `stg_shopify__abandoned_checkout_discount_code` tables. If the user is leveraging the union feature, this could change data values, so would recommend a `dbt run --full-refresh` in this case. 
```

Comment:
A full refresh is really only needed if there is an incremental model. Since this is not an incremental model we should remove that blurb. 
```suggestion
- Added `source_relation` to the `partition_by` clauses that determine the `is_most_recent_record` in the `stg_shopify__metafield` and `stg_shopify__abandoned_checkout_discount_code` tables. If the user is leveraging the union feature, this could change data values.
```

Reply:
Done!